exla/test/exla/device_memory_sharing_test.exs

```diff
@@ -27,14 +27,13 @@ defmodule EXLA.DeviceMemorySharingTest do
   end
 
   @tag :cuda_required
-  test "ipc handles don't crash the runtime when :local mode is selected" do
-    assert {:error, ~c"Invalid pointer size for selected mode."} ==
+  test "invalid ipc handles don't crash the runtime" do
```

Comment:
This test was breaking locally (I guess I hadn't run it on my desktop since the update :stuck_out_tongue:)